### PR TITLE
docs(pipelines): fix link for originarl API document in README

### PR DESCRIPTION
**Pull request by mazyu36**
Fixed a broken link.

### Checklist
- [x] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk-lib/pipelines/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ CDK container assets for ECS clusters instead.
 
 Give the CDK Pipelines way of doing things a shot first: you might find it does
 everything you need. If you need more control, or if you need `v2` support from
-`aws-codepipeline`, we recommend you drop down to using the `aws-codepipeline` 
+`aws-codepipeline`, we recommend you drop down to using the `aws-codepipeline`
```
Comment (ChakshuGupta13) [inline packages/aws-cdk-lib/pipelines/README.md:16]: this change is unrelated and unnecessary (for commit objective), right?

Comment (mazyu36) [reply packages/aws-cdk-lib/pipelines/README.md]: @ChakshuGupta13 
Thank you. Unnecessary spaces were automatically deleted.
I reverted it.​​​​​​​​​​​​​​​​

**Review by ChakshuGupta13 — APPROVED** [review]

**Review by scanlonp — APPROVED** [review]
Nice update, appreciate it!

**Comment by mazyu36** [comment]
@Mergifyio update

**Comment by mazyu36** [comment]
@scanlonp 
Thank you for the approval. The following error occurred, so I referred to the solutions from past PRs and merged main into this branch.
Could you please approve again?
Also, since I don't fully understand the error content, please let me know if my solution was incorrect.

IMG_3319

**Review by scanlonp — APPROVED** [review]
